src/main/java/org/graylog/integrations/aws/resources/KinesisSetupResource.java

```diff
@@ -62,6 +64,15 @@ public KinesisSetupResource(CloudWatchService cloudWatchService, KinesisService
     public KinesisNewStreamResponse createNewKinesisStream(@ApiParam(name = "JSON body", required = true) @Valid @NotNull
                                                                    KinesisNewStreamRequest request) throws InterruptedException {
 
+        // Record the fact that a particular user agreed to create AWS resources.
```

Comment:
Maybe wording such as `"This has been recorded as the listed user has accepted the responsibility in potential incurring associated cost(s)."` 
instead of `"This has been recorded, since AWS resources that will cost money are being created at the sole request of this user"`